.github/workflows/release-candidate.yaml

```diff
@@ -15,121 +15,90 @@ env:
+      - name: Commit changes
+        uses: EndBug/add-and-commit@v9
         with:
-          owner: husarion
-          repo: panther-docker
-          github_token: ${{ secrets.GH_PAT }}
-          workflow_file_name: update-tags-in-compose.yaml
-          ref: ${{ env.RC_BRANCH_NAME }}
-          client_payload: |
-            {
-              "branch_name": "${{ env.RC_BRANCH_NAME }}",
-              "image_tag": "humble-${{ env.RC_BRANCH_NAME }}"
-            }
+          message: Update tags in compose files
+          author_name: action-bot
+          author_email: action-bot@action-bot.com
+          new_branch: ${{ env.RC_BRANCH_NAME }}
 
-  build_and_push_docker_images:
-    name: Build panther docker images
-    runs-on: ubuntu-22.04
-    needs:
-      - update_tags_in_compose
-    steps:
-      - name: Trigger repository build workflow
-        uses: convictional/trigger-workflow-and-wait@v1.6.1
+      - name: Build Docker
+        uses: ./.github/workflows/build-docker.yaml
         with:
-          owner: husarion
-          repo: panther-docker
-          github_token: ${{ secrets.GH_PAT }}
-          workflow_file_name: ros-docker-image.yaml
-          ref: ${{ env.RC_BRANCH_NAME }}
-          client_payload: |
-            {
-              "panther_codebase_version": "${{ env.RC_BRANCH_NAME }}",
-              "build_type": "development",
-              "target_distro": "humble"
-            }
+          build_type: development
 
-  build_and_publish_rpi_image:
-    name: Build panther system image
-    needs:
-      - update_tags_in_compose
+  os_image:
+    name: OS image
     runs-on: ubuntu-22.04
     steps:
-      - name: Trigger repository build workflow
-        uses: convictional/trigger-workflow-and-wait@v1.6.1
+      - name: Create new branch
+        uses: GuillaumeFalourd/create-other-repo-branch-action@v1.5
+        with:
+          repository_owner: husarion
+          repository_name: panther-rpi-os-img
+          new_branch_name: ${{ env.RC_BRANCH_NAME }}
+          new_branch_ref: ros2-devel
+          access_token: ${{ secrets.RAFAL_ACCESS_TOKEN}}
+
+      - name: Build OS image
+        uses: convictional/trigger-workflow-and-wait@v1.6.5
         with:
           owner: husarion
           repo: panther-rpi-os-img
-          github_token: ${{ secrets.GH_PAT }}
+          github_token: ${{ secrets.RAFAL_ACCESS_TOKEN }}
           workflow_file_name: build_and_deploy_image.yaml
           ref: ${{ env.RC_BRANCH_NAME }}
           client_payload: |
             {
               "dev_image": "true",
-              "panther_codebase_version": "${{ env.RC_BRANCH_NAME }}",
+              "husarion_ugv_version": "${{ env.RC_BRANCH_NAME }}",
               "image_tag": "${{ github.event.inputs.version }}"
             }
 
-  build_and_publish_rpi_flash_os_image:
-    name: Build panther flash OS image
-    if: ${{ fromJSON(github.event.inputs.automatic_mode) == true }}
```

Comment:
Why are we removing this condition?

Reply:
Because there is not such input as `automatic_mode` ;)